docs/user/getting-started.md

```diff
@@ -100,38 +100,42 @@ entry
 [11355980 rows x 1 columns]
 ```
 
-### Uproot
+### uproot
+
+Instead of a rucio dataset, here we will use a file directly available over `https`,
+and a slightly more complex query, and we'll ask for the data to be locally downloaded
+so we can access the files directly.
 
 ```python
-import pandas as pd
 from func_adl_servicex import ServiceXSourceUpROOT
-
-dataset_name = "data15_13TeV:data15_13TeV.00282784.physics_Main.deriv.DAOD_PHYSLITE.r9264_p3083_p4165_tid21568807_00"
-src = ServiceXSourceUpROOT(dataset_name, "CollectionTree")
-data = src.Select("lambda e: {'JetPT': e['AnalysisJetsAuxDyn.pt']}") \
-    .AsParquetFiles('junk.parquet') \
-    .value()
-df = pd.read_parquet(data[0])
-print(df)
+import servicex
+import awkward as ak
+
+dataset_name = [
+    "https://xrootd-local.unl.edu:1094//store/user/AGC/nanoAOD/nanoaod15.root"
```

Comment:
While I am not aware of any intention of moving this file, I am wondering whether in the longer term it would be safer to use [scikit-hep-testdata](https://github.com/scikit-hep/scikit-hep-testdata/) (or an IRIS-HEP version of such a repository). A potential issue of the file disappearing would be alleviated by CI of course that could catch it when it happens.

Reply:
Strong agree here.